colm/tableflux/expected/query01-expected.txt

```diff
@@ -0,0 +1,5 @@
+table | _time                | surface_degrees
```

Comment:
I would prefer not to checkin the generated flux files as they are subject to change and will just bloat diffs going forward. Can you run it and look at the output?

Reply:
I need to add some examples demonstrating the approach. Right now those tests are serving as both test and example.

Reply:
Ok, I can understand that reasoning. I wonder if we can add a new commit with the generated files so it is easier for me to review them and then we can remove that commit when we go to merge? If I generate the files locally and look at them, I think it will be too hard for me to make comments on them.

Reply:
Oh I understand you want use the GitHub comment/review features. Hmm, I wonder if GitHub will keep those comments on deleted commits around.

Seems like a useful GitHub feature in general ... to add the ability to comment on the output of a program, without checking that stuff in.

Maybe I just add them to the PR as attachments.

Reply:
I believe GitHub will keep review comments around even if the commit is deleted. Can you add them as a separate commit? I'm having difficulty with reviewing it as attachments.